src/main/java/ConsoleCommunicator.java

```diff
@@ -17,99 +17,113 @@ public void run() {
         }
     }
 
-    public String chooseGame() {
-        askGameMode();
-        String choice = takeHumanInput();
-        return gameLogic.findGameMode(choice);
-    }
-
-    public String takeHumanInput() {
-        return consoleDisplay.takeInput();
-    }
-
     public void runComputerGame() {
+        String statusMessage;
         displayGrid();
         while (gameOngoing()) {
             if (itIsHumanTurn()) {
```

Comment:
When you have a HumanPlayer and a ComputerPlayer class, see how you could refactor this so it doesn't need to check whether the player is human of CPU. Things to research: "duck typing" and "polymorphism".